pkg/proxy/proxy.go

```diff
@@ -76,8 +76,16 @@ func ConvertHostsToRedirectURLs(hosts []string, sso *apiv1.SSO) []string {
 	return redirectURLs
 }
 
-func buildName(name string, namespace string) string {
-	return fmt.Sprintf("%s-%s", namespace, name)
+func buildName(resourceName string, suffix string) string {
+	name := resourceName
+	if len(resourceName) > 62-len(suffix) {
```

Comment:
nit: I think this condition will be true when the length of the `suffix` is grater than 62 charts. I would consider checking also the length of the suffix to avoid the overrun.